.github/workflows/build.sh

```diff
@@ -32,6 +32,9 @@ echo "::endgroup::"
 
 echo "::group::Build wheel"
   clean_code
+  if [[ "$PLAT" == "universal2" ]]; then
+    export MB_PYTHON_OSX_VER="10.10"
```

Comment:
Can this be moved to be next to the other `MB_PYTHON_OSX_VER` export?

https://github.com/python-pillow/pillow-wheels/blob/c5def31b0f72ad620c6a28fa5e9daa4833d861f1/.github/workflows/build.sh#L17-L18

Reply:
It's not quite as simple as that - https://github.com/radarhere/pillow-wheels/commit/82dba2b0d64f2667f7ca59d712f5016ef9ac8dae / https://github.com/radarhere/pillow-wheels/actions/runs/4024580904/jobs/6916784545

Reply:
That PR has been merged, so I've removed this code.